src/components/Svg/Check.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+
+export function Check({ className, alt = 'Check' }: { className?: string; alt?: string }): ReactElement {
+    return (
+        <svg
+            className={classNames(className, 'Svg', 'Svg__check')}
```

Comment:
Should this have `stroke-current` or `fill-current` too? The Svg component generator should have added that automatically so curious if you removed it for a reason here.